validator/client/src/main/java/tech/pegasys/teku/validator/client/loader/ValidatorLoader.java

```diff
@@ -110,8 +110,9 @@ private Signer createSigner(
         keyProviders.add(new YamlValidatorKeyProvider());
       }
 
+      KeystoreLocker keystoreLocker = new KeystoreLocker();
       if (config.getValidatorKeystorePasswordFilePairs() != null) {
-        keyProviders.add(new KeystoresValidatorKeyProvider());
+        keyProviders.add(new KeystoresValidatorKeyProvider(keystoreLocker));
```

Comment:
Probably just create `KeystoreLocker` inline as there's no need to create it if we don't go on to create a `KeystoresValidatorKeyProvider`.

Reply:
Done.